CHANGES

```diff
@@ -1,11 +1,14 @@
-0.3.7 (unreleased)
+0.3.8 (unreleased)
+------------------
+
+- New tool: JPL Horizons service to obtain ephemerides, orbital elements, and
+  state vectors for Solar System objects.
+
+
+0.3.7 (2018-01-25)
 ------------------
 
 - New tool: Exoplanet Orbit Catalog, NASA Exoplanet Archive [#771]
-- MAST: Added convenience function to list available missions. [#947]
```

Comment:
Could you please do a rebase to remove these unrelated changes?  I'll be able to do a full review later next week.

Reply:
Thanks! Please let me know if the latest commit does what you want. If not please be more specific - I am not yet really familiar with the term 'rebase`. Do you want me to remove all unrelated changes? Sorry.